src/Transformation/Transformations/Date.php

```
@@ -0,0 +1,28 @@
+<?php
+/* Copyright (c) 2018 Nils Haagen <nils.haagen@concepts-and-training.de> Extended GPL, see docs/LICENSE */
+
+namespace ILIAS\Transformation\Transformations;
+use ILIAS\Transformation\Transformation;
+
+/**
+ * Transform value to php \DateTime
+ */
+class Date implements Transformation {
+
+	/**
+	 * @inheritdoc
+	 */
+	public function transform($from) {
+		if($from) {
+			return new \DateTime($from);
+		}
+		return '';
```

This is bad. The transformation promises to return a \DateTime, this ain't one. Please just throw an `\InvalidArgumentException` if the input to `transform` is not as expected.